lib/revision_feedback_service.rb

```diff
@@ -1,12 +1,18 @@
 # frozen_string_literal: true
 class RevisionFeedbackService
-  def initialize(revision)
+  # Imports the features from the revisions if revisions is nil otherwise uses the features passed
+  def initialize(features = nil, revision = nil)
```

Comment:
If we're not going to use this with a revision as the argument at this point, then we may as well remove that branch and just take a features hash as the required argument.

For interfaces that need to be flexible in terms of taking one kind of argument or another, it's nicer to use keyword arguments instead of positional arguments. Otherwise, using the revision option would require passing nil as the first argument.